Overview.md

```diff
@@ -0,0 +1,114 @@
+
+# General Overview of Microsoft Azure Event Hubs client for Java
+
+This Java client library for Azure Event Hubs allows for both sending events to and receiving events from an Azure Event Hub. 
+
+An **event publisher** is a source of telemetry data, diagnostics information, usage logs, or other log data, as 
```

Comment:
We use publisher/publishing and sender/sending interchangeably. Could that be a source of confusion? Should we just stick to sender/sending?

Reply:
I agree with the feedback.
overview.md is a split from readme.md - not added newly. 
we eventually want to remove this file as - per the sdk fundamentals teams' guidelines - we need to de-dup our documentation from GitHub repos vs standard documentation channels.

hence, I am deferring the comments in overview.md file for now - to get the main issue of setting EnvironmentVariables for CITs merged.